src/ui/components/Hero/styles.scss

```diff
@@ -40,6 +40,19 @@
   }
 }
 
+.HomeHeroBanner--experiment-is-on {
```

Comment:
I think this should have a more meaningful name, explaining what it does rather than why it is displayed.

Reply:
hmm I thought it would be better to generalize it but sure I can update..

Reply:
My idea is: if we keep this style, then we'll have nothing to update here once we get rid of the experiment.

Reply:
I see ..that makes sense